tests/features/news.feature

```diff
@@ -30,8 +30,6 @@ Feature: News content creation
     And I fill in "Short title" with "Shorter title"
     When I press "Save"
     Then I should see "My News item"
-    And I should see "Navi title"
-    And I should see "Shorter title"
```

Comment:
If we configure this field hidden, we should check it, as ` And I should not see "Shorter title"`, isn't it?

Reply:
Not really necessary I guess. It's standard behaviour...

Reply:
I actually agree with @voidtek on this one 😃

Reply:
Let's convert this to "I should not see" 😄